test/unit/explain.test.ts

```diff
@@ -51,35 +51,35 @@ describe('class Explain {}', function () {
     });
   });
 
-  describe('parseTimeoutOptions()', function () {
+  describe('resolveExplainTimeoutOptions()', function () {
     const cursor = new FindCursor(
       new MongoClient('mongodb://localhost:27027'),
       MongoDBNamespace.fromString('foo.bar'),
       {},
       {}
     );
 
-    it('parseTimeoutOptions()', function () {
+    it('resolveExplainTimeoutOptions(no arguments)', function () {
```

Comment:
as long as we're improving this, can we update all these to actually be meaningful? e.g. `does not set timeout or explain when no arguments are provided`, `sets only timeout when only timeoutMS is provided`